src/components/tutorialEditor/index.tsx

```diff
@@ -135,12 +302,148 @@ const TutEditor: React.FC = (): JSX.Element => {
+                        <StyledButton
+                          style={{
+                            background:
+                              value.type === "text"
+                                ? "rgba(256, 256, 256, 0.2)"
+                                : "none",
+                            userSelect: "none"
+                          }}
+                          onClick={(): void => {
+                            changeType("text", index);
+                          }}
+                        >
+                          Text
+                        </StyledButton>
+                        <StyledButton
+                          style={{
+                            background:
+                              value.type === "code"
+                                ? "rgba(256, 256, 256, 0.2)"
+                                : "none",
+                            userSelect: "none"
+                          }}
+                          onClick={(): void => {
+                            changeType("code", index);
+                          }}
+                        >
+                          Code
+                        </StyledButton>
+                        <UpDownContainer>
+                          <Arrow
+                            icon={chevronUp}
+                            onClick={(): void => {
+                              changeOrder("up", index);
+                            }}
+                          />
+                          <Arrow
+                            icon={chevronDown}
+                            onClick={(): void => {
+                              changeOrder("down", index);
+                            }}
+                          />
+                        </UpDownContainer>
+                        <TextInput
+                          value={content[index].value}
+                          style={{
+                            display:
+                              content[index].type === "text" ? "block" : "none"
+                          }}
+                          type="text"
+                          placeholder="Type text here..."
+                          onChange={(
+                            event: React.ChangeEvent<HTMLInputElement>
+                          ): void => {
+                            handleTextChange(index, event);
+                          }}
+                        />
+                        <AceEditor
+                          value={content[index].value}
+                          style={{
+                            position: "relative",
+                            marginTop: "0px",
+                            height: "82px",
+                            width: "100%",
+                            backgroundColor: background,
+                            fontFamily: "Source Code Pro",
+                            display:
+                              content[index].type === "code" ? "block" : "none"
+                          }}
```

Comment:
It might be worth storing the styles as an object and then pass then into a style prop rather than making them inline. I'm just thinking of ways to keep the JSX cleaner since that will be looked at more than the CSS.